Services/Config/Disscount.js

```diff
@@ -0,0 +1,23 @@
+module.exports = (object) => {
+  let totally = 0;
+
+  if (object.code === 'pants') {
+    if (object.qty % 2 === 0) {
+      totally += parseInt((object.qty / 2), 10) * object.price;
+      return totally;
+    }
+    totally += (parseInt((object.qty / 2), 10) * object.price) + object.price;
+    return totally;
+  }
+
+  if (object.code === 'tshirt') {
+    if (object.qty >= 3) {
+      totally += object.qty * 19;
```

Comment:
- Is not `object.price` available on your input?
- What will happen if the price for t-shirts is changed on the database?

Reply:
Object.price is into the database, in the case of shirt I used a magic number "19" I mean the discount that shirt has when someone buy 3 or more.

Reply:
Yeah, but if the price for t-shirt is changed, you'll come back and update that magic number?

I mean, that's not even easier than `(object.price - 1)` right?

Reply:
Yeah! You right, I'll fix it up. Thanks